### PR TITLE
commit 1

### PR DESCRIPTION
**Pull request by NikolenkoOlga**

### CODE REVIEW

File: Task_1/MaxSequence.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        }
+
+        /// <summary>
+        /// Method for counting the maximum number of identical consecutive elements in a string
+        /// </summary>
+        /// <param name="str">string for processing</param>
+        /// <returns> The maximum number of identical consecutive elements in a string </returns>
+        public int CalculateMaxNumber()
+        {
+
```
Comment (IgorKheidorov) [inline Task_1/MaxSequence.cs]: Code style, empty string

File: Task_1/MaxSequence.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            }
+            if (counterOfChars > maxSequence)
+            {
+                maxSequence = counterOfChars;
+            }
+            return maxSequence;
+        } 
+     }
+}
+
```
Comment (IgorKheidorov) [inline Task_1/MaxSequence.cs]: Empty strings

File: Task_1/MaxSequence.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        /// Method for counting the maximum number of identical consecutive elements in a string
+        /// </summary>
+        /// <param name="str">string for processing</param>
+        /// <returns> The maximum number of identical consecutive elements in a string </returns>
+        public int CalculateMaxNumber()
+        {
+
+            int counterOfChars = 1;
+            int maxSequence = 0;
+            for (int i = 0; i < inputString.Length - 1; i++)
```
Comment (IgorKheidorov) [inline Task_1/MaxSequence.cs]: Logical error

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+*.VC.db
+*.VC.VC.opendb
+
+# Visual Studio profiler
+*.psess
+*.vsp
+*.vspx
+*.sap
+
+# TFS 2012 Local Workspace
```
Comment (AlexanderSoroka) [inline .gitignore:93]: you don't need this as you don't use tfs

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+*.mm.*
+AutoTest.Net/
+
+# Web workbench (sass)
+.sass-cache/
+
+# Installshield output folder
+[Ee]xpress/
+
+# DocProject is a documentation generator add-in
```
Comment (AlexanderSoroka) [inline .gitignore:128]: do you use DocProject?

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+.fake/
+
+# JetBrains Rider
+.idea/
+*.sln.iml
+
+# CodeRush
+.cr/
+
+# Python Tools for Visual Studio (PTVS)
```
Comment (AlexanderSoroka) [inline .gitignore:259]: Python????

File: Task_1/MaxSequence.cs [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+                        maxCount = currentCount;
+                    }
+                    currentCount = 1;
+                }
+            }
+            if (currentCount > maxCount)
+            {
+                maxCount = currentCount;
+            }
+            return maxCount;
```
Comment (AlexanderSoroka) [inline Task_1/MaxSequence.cs:42]: if maxSequence is empty it will return wrong value

File: Task_1/Properties/AssemblyInfo.cs [hunk 8]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Общие сведения об этой сборке предоставляются следующим набором
```
Comment (AlexanderSoroka) [inline Task_1/Properties/AssemblyInfo.cs:5]: Russian is not acceptable

**Review by AlexanderSoroka — APPROVED** [review]
Approved due to deadline, not accepted as it's extremely dirty

**Pull request merged**